UHResidentInformationAPI/V1/Gateways/UHGateway.cs

```diff
@@ -21,7 +24,43 @@ public List<ResidentInformation> GetAllResidents(string houseReference, string r
 
         public ResidentInformation GetResidentById(string houseReference, int personReference)
         {
-            return null;
+            var databaseRecord = _uHContext.Persons.FirstOrDefault(p => p.HouseRef == houseReference && p.PersonNo.Equals(personReference));
+
+            if (databaseRecord == null) return null;
+
+            var telephoneNumberForPerson = _uHContext.TelephoneNumbers.Where(t => t.ContactID == databaseRecord.PersonNo).ToList();
+
+            var emailAddressForPerson =
+                _uHContext.EmailAddresses.Where(c => c.ContactID == databaseRecord.PersonNo).ToList();
+
+            var addressForPerson =
+                _uHContext.Addresses.FirstOrDefault(a => a.HouseRef == databaseRecord.HouseRef);
```

Comment:
Should this be ordered by something before we pick the first record? Maybe so that we return the address which has been most recently modified?

Reply:
That's true.
On hindsight, used this method based on the IDE suggestion when I attempted to use `where` to query it.
Could look into making sure the most up to date address entry is used. Maybe ordering it by a column that tracks the last modified date.

Reply:
Changes made in 1a556eb